### ISSUE TITLE
Request - network activation

### ISSUE THREAD

**Issue by pixeltester** [teakhanirons/ftpeverywhere#1]
Would it be possible for you to add some form of network check to activate the Vita's wireless connection when idle?

This would be useful for example with a newly powered on system as wireless won't connect unless something with a network task begins.

**Comment by teakhanirons** [comment]
Yes, I also got this request from someone else. I was thinking about a plugin to make small network requests every 5 seconds or so in a thread or something similar but got distracted because of other projects. As a workaround, I found launching an closing any app to work.
